generics/middleware/authenticator.js

```diff
@@ -65,7 +65,7 @@ module.exports = async function (req, res, next, token = '') {
 		return
 	}
 
-	let internalAccessApiPaths = ['/templates/bulkCreate', '/projectAttributes/update']
+	let internalAccessApiPaths = ['/templates/bulkCreate', '/projectAttributes/update', '/userProjects/addStory']
```

Comment:
@MallanagoudaB Please move this to guest url and  make sure it is working without any auth keys. if require we can make it internal later